static/app/views/metrics/metricListItemDetails.tsx

```diff
@@ -122,11 +124,23 @@ export function MetricListItemDetails({
         <MetricName>
           {/* Add zero width spaces at delimiter characters for nice word breaks */}
           {formatMRI(metric.mri).replaceAll(/([\.\/-_])/g, '\u200b$1')}
+          {isDuplicateWithDifferentUnit && (
+            <Disclaimer textColor="yellow400">
+              <IconWrapper>
+                <IconWarning color="yellow400" size="xs" />
+              </IconWrapper>
+              {t(
+                'Metrics with the same name and different units were detected. Disable any unwanted ones in settings.'
```

Comment:
Can we link out to the relevant page in settings here?

Reply:
the cog icon next to it will take you where you need to go:

![CleanShot 2024-05-27 at 18 32 15](https://github.com/getsentry/sentry/assets/9060071/7bb1d54e-881e-4919-9bd2-0c7e69424a74)

Reply:
let me try with blue color to make it a bit more visible